python/py-musicbrainzngs/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-musicbrainzngs
+version             0.7.1
+revision            0
+
+categories-append   multimedia
+license             BSD
+platforms           darwin
+supported_archs     noarch
+
+description         Python bindings for Musicbrainz' NGS webservice
+long_description    This library implements webservice bindings for the Musicbrainz NGS site, \
+                    also known as /ws/2 and the Cover Art Archive.
+
+homepage            https://python-musicbrainzngs.readthedocs.io/
+
+maintainers         nomaintainer
+
+checksums           rmd160  81735f581536f9b6ffaa5606b53732d29c7cc24e \
+                    sha256  ab1c0100fd0b305852e65f2ed4113c6de12e68afd55186987b8ed97e0f98e627 \
+                    size    117469
+
+python.versions     37 38 39
+
+if {${name} ne ${subport}} {
+    livecheck.type  none
```

Comment:
why don't you now not need the `setuptools` dependency anymore?

Reply:
@reneeotten it is typo, thanks